appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -767,26 +767,93 @@
         (*:addParent (KawaEnvironment:getCurrent) *test-environment*)
         (set! *test-global-var-environment* (gnu.mapping.Environment:make 'test-global-var-env)))))
 
-(define-syntax foreach
-  (syntax-rules ()
-    ((_ lambda-arg-name body-form list)
-     (yail-for-each (lambda (lambda-arg-name) body-form) list))))
-
 
-(define-syntax forrange
+;; Note: (Jeff Schiller) The macro below is intentially unhygienic. We
+;; need to make sure that if there is a *yail-break* form inside
+;; bodyform that it does not get shadowed by the macro (which it would
+;; if this was a hygienic macro).
+
```

Comment:
What a nice way to fix this!!

Reply:
Now all I need to do is spell intentionally correctly.